Features/Wallet/Package.swift

```diff
@@ -0,0 +1,52 @@
+// swift-tools-version: 6.0
+
+import PackageDescription
+
+let package = Package(
+    name: "Wallet",
+    platforms: [
+        .iOS(.v17)
+    ],
+    products: [
+        .library(
+            name: "Wallet",
```

Comment:
```suggestion
            name: "WalletTab",
```

`Wallet` is very generic name, could be an issue with `Wallet` as the model. Maybe for big features use it as `WalletTab`, `SwapTab` as a feature name?